src/main/resources/templates/aeolus/java/plain_gradle.yaml

```diff
@@ -0,0 +1,27 @@
+api: v0.0.1
+metadata:
+    name: java-gradle
+    id: java-windfile
+    description: Build Plan for a java Exercise
+    author: Andreas Resch
+    docker:
+        image: ls1tum/artemis-maven-template
+        tag: java17-20
+        volumes:
+            - ${WORKDIR}:${WORKDIR}
+            - ${TMPDIR}:${TMPDIR}
+        parameters:
+            - --cpus
+            - '"2"'
+            - --memory
+            - '"2g"'
+            - --memory-swap
+            - '"2g"'
+            - --pids-limit
+            - '"1000"'
+actions:
+    - name: gradle
```

Comment:
The SCA step is missing here, is that intentional?

Reply:
LocalCI doesn't support SCA (yet)

Reply:
So this template is currently only for use in localci? The idea of windfiles (if I understood that correctly) includes all supported build systems.

Reply:
yes, this PR simply adds UI elements and some other preparations for the eventual integration for multiple platforms